service/store/v1/store.go

```diff
@@ -169,14 +163,36 @@ func (s *Store) removeExistingRSAKeys() error {
 	return createFolder(s.keyFolderPath)
 }
 
+func readFile(fileName string) ([]byte, error) {
+	content, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		return nil, err
+	}
+
+	stats, err := os.Stat(fileName)
+	if err != nil {
+		return nil, err
+	}
+	if stats.Mode().Perm() != fs.FileMode(0600) {
+		return nil, fmt.Errorf("file \"%s\" has not the right permission, expected 0600", fileName)
+	}
```

Comment:
Not sure that check is correct. can we say `>=`. Validating against insufficient permission should be an error, but if the file have more permission that expected, the program should probably not fail?

Reply:
About not enough permissions, I think the read and write are going to fail anyway, so I would say that's already ok.

> If the file have more permission that expected, the program should probably not fail.

This is measure similar to the one of SSH has put in place, so we can ensure sensitive files are not open to the world.

What do you think about this ?